phoenix-core/src/main/java/org/apache/phoenix/coprocessor/BaseScannerRegionObserver.java

```diff
@@ -88,6 +88,8 @@
     public static final String INDEX_REBUILD_PAGING = "_IndexRebuildPaging";
     // Index verification type done by the index tool
     public static final String INDEX_REBUILD_VERIFY_TYPE = "_IndexRebuildVerifyType";
+    public static final String INDEX_REBUILD_DISABLE_LOGGING_VERIFY_TYPE =
```

Comment:
You would need to rebase with latest 4.x I suppose.

Reply:
Done.